src/microbe_stage/Microbe.cs

```diff
@@ -2108,6 +2110,82 @@ private Transform GetNewPhysicsRotation(Transform transform)
         return new Transform(new Basis(slerped), transform.origin);
     }
 
+    /// <summary>
+    ///   Adds torque to reach the targeted rotation
+    /// </summary>
+    private void ApplyPhysicsRotation(PhysicsDirectBodyState physicsState)
+    {
+        var transform = physicsState.Transform;
+        var target = transform.LookingAt(LookAtPoint, new Vector3(0, 1, 0));
+        if (transform == target)
+            return;
+
+        var strength = Constants.CELL_BASE_THRUST + Constants.CELL_BASE_THRUST * (HexCount/2);
+        var maxVelocity = 5.0f;
+
+        var distance = target.basis.GetEuler().y - transform.basis.GetEuler().y;
+        distance = (float)(distance * (180/Math.PI));
+        if (distance > 180) {
+            distance = -360 - distance;
+        } else if (distance < -180) {
+            distance = 360 + distance;
+        }
```

Comment:
I assumed by default that `distance` was a measure in meters. You might want to go for a more specific name if this deals with angles.

Also, why changing to degrees?

Reply:
What other name for distance would you suggest? I cant think of a good one.
I changed it to degrees, because it was easier for me to work with. But I can change it back if you want.

Reply:
Well, I think I'd go for `angleToTarget`? But this is just my mind as a reviewer, you should make sure to pick a name you find clear as the very person who coded this PR. Take your time to come up with a name your really like.

Reply:
As for degrees, the style doc does not mention this, so I guess I can't really tell for sure you should do this. I just know that, as far as I recall, the rest of the code uses radians, which are furthermore scientific system units. So I think you should use radians indeed.